cljfmt/project.clj

```diff
@@ -21,5 +22,27 @@
                                  :optimizations :none}}}
               :test-commands
               {"dev" ["node" "target/out/tests.js"]}}
+  :native-image
+  {:name "cljfmt"
+   :opts
+   ["--verbose"
+    "-H:+ReportExceptionStackTraces"
+    "-J-Dclojure.spec.skip-macros=true"
+    "-J-Dclojure.compiler.direct-linking=true"
+    "-H:ReflectionConfigurationFiles=reflection.json"
+    "--initialize-at-build-time"
+    "-H:Log=registerResource:"
+    "--verbose"
+    "--no-fallback"
+    "--no-server"
+    "-J-Xmx3g"]}
   :profiles
-  {:provided {:dependencies [[org.clojure/clojurescript "1.10.520"]]}})
+  {:uberjar
+   {:source-paths
+    ["standalone"]
```

Comment:
Can you put `["standalone"]` on the same line as `:source-paths`. Typically we shouldn't linebreak keys unless we run out of horizontal room.